server/planning/planning.py

```diff
@@ -69,13 +80,158 @@ class PlanningService(superdesk.Service):
     },
 
     # Planning Details
-    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16.5.1
+    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16
     # probably can skip this subsection, although its documented in iptc impl guide this way
-    'planning_details': {
+    'planning_item': {
         'type': 'dict',
         'schema': {
-            # TODO: fill in from NewsML-G2 version 2.21 spec
-        }
+            # See IPTC-G2-Implementation_Guide 16.1
+            'item_meta': {
```

Comment:
imo it's easier to work with flat structure for both item meta and content meta, when you do some filtering etc. so you could do search for `headline:foo` rather than `item_meta.headline:foo` etc.